ts/sagas/profile.ts

```diff
@@ -328,13 +329,17 @@ function* checkStoreHashedFiscalCode(
     isDifferentFiscalCodeSelector,
     profileLoadSuccessAction.payload.fiscal_code
   );
-  // the current logged user has a different fiscal code from the stored hashed one
-  if (checkIsDifferentFiscalCode === true) {
-    yield put(differentProfileLoggedIn());
-  }
+
   yield put(
```

Comment:
@Undermaken `profile` test suite broke if I put this in the first `if` case, wouldn't it be better to avoid a call to `AsyncStorage` when fiscal code is the same?